rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -353,7 +353,9 @@ void gs_frame::keyPressEvent(QKeyEvent *keyEvent)
 				gui_log.warning("The video provider could not release the image sink. A sink with higher priority must have been set.");
 			}
 
-			rsx::overlays::queue_message(tr("Recording stopped: %0").arg(QString::fromStdString(m_video_encoder->path())).toStdString());
+			ensure(m_video_encoder->path().find(fs::get_config_dir()) != m_video_encoder->path().npos);
```

Comment:
starts_with

Reply:
Btw, we don't really need npos. npos is simply std::size_t(-1) which is the same as writing umax.

Reply:
Done. I also added a sound by the way.